pkg/handlers/handlers_test.go

```diff
@@ -390,3 +390,27 @@ func BenchmarkHandleTar(b *testing.B) {
 		assert.NoError(b, err)
 	}
 }
+
+func TestHandleLargeHTTPJson(t *testing.T) {
+	resp, err := http.Get("https://raw.githubusercontent.com/ahrav/nothing-to-see-here/main/md_random_data.json.zip")
+	assert.NoError(t, err)
```

Comment:
i learned recently that `NoError` returns a value that you can use to bail on the rest of the test if appropriate

Reply:
Do you prefer that over an explicit err != nil + t.Fatal?

Reply:
I ask because we also use the if err != nil + t.Fatal approach, and I want to ensure consistency if possible. I don't have a strong opinion either way, so i'll go with whatever people feel is more appropriate.